package.json

```diff
@@ -76,7 +76,7 @@
   },
   "devDependencies": {
     "@adobe/jsonschema2md": "^7.1.5",
-    "@fluencelabs/installation-spell": "^0.5.7",
+    "@fluencelabs/installation-spell": "^0.5.9",
```

Comment:
did installation spell air change? if yes - it needs to be manually updated (unstil new release of aqua this is still part of the process for now)

Reply:
ok it didn't change, I checked